tests/test_game.py

```diff
@@ -1,12 +1,15 @@
-from unittest import TestCase
+import io
+import sys
+from functools import partial
+from unittest import TestCase, mock
 
 from card import Card
 from game import Game
 
 
 class GameTestCase(TestCase):
     @classmethod
-    def setUpClass(cls):
+    def setUp(cls):
```

Comment:
While you are here, you likely want to add in `super().setUp()` as the first line of the method. Additionally, `setUp` is not a `classmethod`. And as such, the first argument should be `self` instead of `cls`